sdk/lib/rtl/image.c

```diff
@@ -289,17 +289,40 @@ RtlImageDirectoryEntryToData(
     if (NtHeader == NULL)
         return NULL;
 
-    if (Directory >= SWAPD(NtHeader->OptionalHeader.NumberOfRvaAndSizes))
-        return NULL;
+#ifdef _WIN64
+    if (NtHeader->OptionalHeader.Magic == IMAGE_NT_OPTIONAL_HDR64_MAGIC)
```

Comment:
Actually... if you call the user mode version of this function, does it recognize images that don't match your process's bitness?

Reply:
https://pastebin.com/v4awm9LE

Test succeeds for me as 32 bit test, so at least the 32 bit version can recognize a 64 bit (in-memory) image

Reply:
![image](https://user-images.githubusercontent.com/610685/98861582-d348ea00-2465-11eb-82dd-58687a59323e.png)
2k3

Reply:
@learn-more Your paste is gone. Can you repaste is somewhere or commit it?

Reply:
Added #3383